docker/server/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
 WORKDIR /app
 
 # Copy everything from source
-RUN git clone https://github.com/helayoty/carbon-aware-sdk.git && cd carbon-aware-sdk && git checkout 55d75c7
+RUN git clone https://github.com/Green-Software-Foundation/carbon-aware-sdk.git && cd carbon-aware-sdk
```

Comment:
With the env vars approach I don't think a fork is needed so I removed it here.

Reply:
Awesome!

Reply:
This is Great. many thanks @rossf7 !